indico/modules/events/agreements/templates/agreement_type_details.html

```diff
@@ -119,23 +119,19 @@
         <div class="agreement-list i-box{% if not (not_sent or pending) %} titled{% endif %}">
             {% if not_sent %}
                 <div class="toolbar">
-                    <div class="group">
-                        <a href="#" class="i-button icon-checkbox-checked arrow js-dropdown" data-toggle="dropdown"></a>
-                        <ul class="dropdown">
-                            <li><a href="#" id="select-new">
-                                {% trans %}All{% endtrans %}
-                            </a></li>
-                            <li><a href="#" id="select-new-none">
-                                {% trans %}None{% endtrans %}
-                            </a></li>
-                        </ul>
-                    </div>
-                    <div class="group">
-                        <a id="send-new" class="i-button icon-mail disabled js-ajax-dialog-send" href="#"
-                           data-href="{{ url_for('.event_agreements_details_send', event, definition=definition.name) }}">
-                            {% trans %}Send new{% endtrans %}
-                        </a>
-                    </div>
+                    <a href="#" class="i-button icon-checkbox-checked arrow js-dropdown" data-toggle="dropdown"></a>
+                    <ul class="dropdown">
+                        <li><a href="#" id="select-new">
```

Comment:
I know you just changed the indentation, but could you move the `<a>...</a>` to their own lines instead of having them in one line with the `<li>`? We never put two tags in the same line like here anywhere else.